src/main/java/uk/gov/register/views/ConsistencyProof.java

```diff
@@ -23,7 +26,9 @@ public String getProofIdentifier() {
 
     @SuppressWarnings("unused, used as jsonproperty")
     @JsonProperty("merkle-consistency-nodes")
-    public List<String> getConsistencyNodes() {
+    public List<HashValue> getConsistencyNodes() {
+        List<String> nodes = consistencyNodes.stream().map(p -> p.toString()).collect(Collectors.toList());
```

Comment:
The `nodes` variable isn't being used and isn't required.

Also this results in `"merkle-consistency-nodes":[{"value":"bbb6942aa14ed87aeb180e9f9bfdeab62bc537222dae0f8ee91433c92c7de48d"}` being returned incorrectly by the endpoint.

If you annotate this method with `@JsonSerialize(using = ToStringSerializer.class)` then you will get the correct result.